lib/src/yaru_selectable_container.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+class YaruSelectableContainer extends StatelessWidget {
+  /// Creates a Image Tile from the image path given in the path property.
+  const YaruSelectableContainer({
+    Key? key,
+    required this.child,
+    this.onTap,
+    required this.currentlySelected,
+    this.borderRadius,
+    this.padding,
+    this.selectionColor,
+  }) : super(key: key);
+
+  // The child which will be selected with [onTap]
+  final Widget child;
+
+  /// Current Value of the imageTile.
+  /// Based on the this value selection of the image can be managed.
+  /// If this value is `true` [Container] border will have color from [Theme.of(context).primaryColor]
+  /// else if the value is `false` the border color will be [Colors.transparent].
+  final bool currentlySelected;
+
+  /// Callback triggered when the [YaruSelectableContainer] is clicked.
+  final VoidCallback? onTap;
+
+  /// Optional custom radius for the corners which defaults to a circular 6.0 [BorderRadius]
+  final BorderRadius? borderRadius;
+
+  /// Optional custom padding for the child which defaults to 6.0 on all sides.
```

Comment:
Maybe it'd be clearer if the default value was specified in the constructor?

Reply:
ah, well if we would stick with double yes. But I thought it would be better to leave the full freedom of using any combination of different roundings at the 4 edges as one wishes. With the BorderRadius as one property I can not default to BorderRadius.circular() because: "The default value of an optional parameter must be constant."

Reply:
nvm, figured it out myself ^^

Reply:
Yes, `BorderRadius.cicular()` has no `const` constructor. You can use: `BorderRadius.all(Radius.circular(6))` but I see you managed to do it in a different way :)